test/Assertions.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace test\eLife\Journal;
+
+use GuzzleHttp\Psr7\UriNormalizer;
+use Psr\Http\Message\UriInterface;
+use function GuzzleHttp\Psr7\uri_for;
+
+trait Assertions
```

Comment:
a bit like the `Utils` class. Name it `UriAssertions` for cohesion?

Reply:
Could do, had though any other assertions could be put here but a trait each is also fine.